examples/lang/erc20/src/lib.rs

```diff
@@ -75,16 +73,16 @@ contract! {
         /// Returns the balance of the given AccountId.
         pub(external) fn balance_of(&self, owner: AccountId) -> Balance {
             let balance = self.balance_of_or_zero(&owner);
-            env.println(&format!("Erc20::balance_of(owner = {:?}) = {:?}", owner, balance));
+            env.println(&format!("Erc20::balance_of(owner = {:?}) = {:?}", owner.0, balance));
```

Comment:
I would rather introduce some getters for the underlying data (e.g. `as_bytes(&self) -> [&u8]` or something like that) instead of using `.0` syntax anywhere in a non-generic library environment. But that's a subjective meaning. Feel free to discuss.

Reply:
The AccountId type from `node_runtime` resolves to `sr25519::Public` didn't implement `Debug for `no_std`. I've just added that on my `aj-ink-types` branch: https://github.com/paritytech/substrate/blob/8cf36aff21f6ca85beedc88e6234076f32a50970/core/primitives/src/sr25519.rs#L50